### ISSUE TITLE
Stow: Buttons text have underlines

### ISSUE THREAD

**Issue by mxhassani** [Automattic/wp-calypso#67249]
### Quick summary

Could be related to this: 

6443-gh-Automattic/themes

4430-gh-Automattic/themes

### Steps to reproduce

1. Apply the Stow theme to a site
2. Open the editor and create a button block and add a link to it

### What you expected to happen

To see the button without an underline:

![image](https://user-images.githubusercontent.com/81776920/187868345-90ef0937-4166-4d0e-972a-3ae77345075d.png)

### What actually happened

The button has an underline:

![image](https://user-images.githubusercontent.com/81776920/187868387-26a3860e-840f-48d5-bf7e-bd2e94473ffd.png)

This doesn't affect the editor.

### Context

5520578-zd-woothemes

### Theme-specific issue?

Yes - Stow

### Reproducibility

Consistent

### Severity

Some (< 50%)

### Available workarounds?

Yes, easy to implement

### Workaround details

CSS:
```
/* Remove underline from buttons */
.entry-content .wp-block-button a{
  text-decoration: none;
}
```

**Comment by sanjeev00733** [comment]
It also affects all the links on the site. Reported in #5526977-zen. 

All links on the site are underlined. Shared the below code to fix as a temp workaround.

```
.entry-content a:link, .entry-content a:active {
   text-decoration: none;
}
```

**Comment by mxhassani** [comment]
> It also affects all the links on the site.

Indeed, it looks like this was the main aim of [the fix](https://github.com/Automattic/themes/pull/6443). But yes, the user ended up asking for all links to be removed. Sent them this:

```
.entry-content .wp-block-button a, .entry-content a, .entry-content a:link, .entry-content a:active {
	text-decoration: none;
}
```

**Comment by zachspears** [comment]
Able to reproduce. Keeping low priority due to low impact.

**Comment by sophiegyo** [comment]
Another report here:

5530292-zen

**Comment by mriyazuddin** [comment]
- [ ] Another one reported here: 5530292-zdn

**Comment by essleeung** [comment]
- [ ] Another report here: 5629107-zd-woothemes

Provided the CSS to remove all underlined links for now.
